ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionBuilderModule.java

```diff
@@ -50,6 +50,7 @@
 public class ExecutionBuilderModule {
 
   private static final Logger LOG = LogManager.getLogger();
+  private static final int HUNDRED_PERCENTS = 100;
```

Comment:
we shouldnt need to define 100%, but if we do, maybe just `HUNDRED_PERCENT`, as the `s` is confusing :)

Reply:
Oh, could you explain this, please, I know you are right, but I don't understand this. Is percent always plural?